README.md

```diff
@@ -46,27 +46,22 @@ There are several steps to importing new content into the prototype:
 0. Tag all the content to taxons using [content
    tagger](https://github.com/alphagov/content-tagger).
 0. Create a list of content base paths in CSV format, with a single column
-   heading `Link`. Save this as `content.csv` in the `bin` folder.
-0. From within `bin`, run `format_finder.py` to fetch format information from
-   the search API.  This will save a file called `link_formats.csv`.
-0. Run `fetch_main_content.py` to scrape the `main` html and store it in
-   `app/content`. This uses the data from the previous step to organise the
-   pages by format (the format directory is ignored by the prototype when
-   rendering pages but this makes it easier to work with). See
+   heading `Link`. Save this as `content.csv` in the `lib/python/data` folder.
+   The `taxonomy:export_base_paths` rake task in `content-tagger` can be used
+   to derive a list of paths from an entire taxonomy.
+0. Run `python lib/python/format_finder.py` to fetch format information from
+   the search API.  This will save a file called
+   `lib/python/data/link_formats.csv`.
+0. Run `python lib/python/fetch_main_content.py` to scrape the `main` html and
+   store it in `app/content`. This uses the data from the previous step to
+   organise the pages by format (the format directory is ignored by the
+   prototype when rendering pages but this makes it easier to work with). See
    `failed_pages.txt` for a list of pages that encountered redirects and 404s.
 0. Run `bundle exec rake` to fetch taxon and content page metadata (e.g. which
```

Comment:
This currently fails because `https://github.com/alphagov/govuk-content-navigation/blob/master/lib/ruby/taxonomy_data.rb#L5` doesn't exist. I think we should delete that line.